.github/workflows/inspektor-gadget.yml

```diff
@@ -1602,10 +1602,7 @@ jobs:
         # GitHub Action.
         az extension add --name aks-preview
 
-        # Let's keep thing in the US to avoid data crossing the Atlantic as
-        # GitHub data centers are in the US:
```

Comment:
Would this mean the latency is now higher while running the tests. So like pulling the gadget images. We already sometimes have issue because of timeouts, would this make our CI even more flaky?

Reply:
Don't they use any kind of geo replication for the container registry?